stock_picking_reverse_auto_check_refund/models/stock_return_picking.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models, fields, api
```

Comment:
```suggestion
# Copyright 2022 Quartile Limited
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).

from odoo import api, fields, models
```

Please run pre-commit.

Reply:
I run the pre-commit and there is a [PR](https://github.com/qrtl/hls-custom/pull/191) for fixing pre-commit issues.